applications/mobu/values.yaml

```diff
@@ -26,9 +26,13 @@ config:
   # Each flock listed will be automatically started when mobu is started.
   autostart: []
 
-  # -- GitHub app integration options.
-  # @default -- See `values.yaml`
-  github: {}
+  # -- Configuration for the GitHub refresh app integration.
+  # @default -- disabled.`
+  githubRefreshApp: {}
+
+  # -- Configuration for the GitHub CI app integration.
+  # @default -- disabled.`
+  githubCiApp: {}
```

Comment:
It would be really nice to be able to get documentation for all of the available options into the Helm `README.md` file by documenting each field, which was one of the other things that pushed me towards a combined configuration file format. But avoiding that for now, maybe add a URL to the mobu documentation?

The `@default` string here seems to be invalid Markdown.